.drone.yml

```diff
@@ -532,7 +532,7 @@ steps:
 - name: upload-packages
   image: grafana/grafana-ci-deploy:1.3.1
   commands:
-  - ./bin/grabpl upload-packages --edition oss
+  - ./bin/grabpl upload-packages --edition oss --packages-bucket grafana-downloads
```

Comment:
because I simplified the logic so that this param is always sent (I assume this is the default in the the code of upload-packages) 

But think it's better to have it spelled out here and makes the python logic simpler